internal/common/voting/election.go

```diff
@@ -20,38 +21,40 @@ func (e *Election) ProposeElection(role shared.Role, method shared.ElectionVotin
 
 // OpenBallot sets the islands eligible to vote.
 func (e *Election) OpenBallot(clientIDs []shared.ClientID) {
-	e.islandsToVote = clientIDs
+	e.voterList = clientIDs
+	//TODO candidate list in each election need to be determined, as it won't always equals to voter list.
+	e.candidateList = e.voterList
 }
 
 // Vote gets votes from eligible islands.
 func (e *Election) Vote(clientMap map[shared.ClientID]baseclient.Client) {
-	for _, island := range e.islandsToVote {
-		e.votes = append(e.votes, clientMap[island].GetVoteForElection(e.roleToElect))
+	for i := 0; i < len(e.voterList); i++ {
```

Comment:
Range based for loop?

Reply:
yeah I was trying to use for loop instead of range to make sure the operating order is the order in voterList